types/injectpromise/index.d.ts

```diff
@@ -0,0 +1,12 @@
+// Type definitions for injectpromise 1.0
+// Project: https://github.com/sullof/injectpromise
+// Definitions by: naeemy <https://github.com/naeemy>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export as namespace injectpromise;
+
+/**
+ * Make a function accepting a callback return a Promise
+ * @param scope - object scope
+ */
+export default function injectpromise(scope: any): (func: (...args: any) => any, ...args: any) => Promise<any>;
```

Comment:
```suggestion
declare function injectpromise(scope: any): (func: (...args: any) => any, ...args: any) => Promise<any>;

export = injectpromise;
```
this is not ESM module:
https://github.com/sullof/injectpromise/blob/master/index.js#L1